yaksh/documentation/moderator_docs/creating_lessons_modules.rst

```diff
@@ -0,0 +1,90 @@
+===================
+Lessons and Modules
+===================
+
+Courses can have lessons and quizzes encapsulated using a module.
+
+	* **What is a lesson?**
+		A lesson can be any markdown text or an embedded video of a particular topic.
```

Comment:
It can also be a combination of both, can't it? Hence - A lesson can be any markdown text with/or an embedded video of a particular topic.